pages/index.js

```diff
@@ -434,511 +448,459 @@ export default class IndexPage extends React.Component {
   async componentDidMount() {
     const response = await Actions.health();
     console.log("HEALTH_CHECK", response);
-    this.addExampleSlates();
-  }
-
-  coolSlates = [
```

Comment:
Did you intend to erase this work that @gndclouds did?

Reply:
no I didn't. really sorry about this, I rebased this morning but not sure why I don't have this in my local repo. I will figure this out

Reply:
No problem! Glad we've caught this now. Also if you view https://filecoin.onrender.com it looks like there was some other work.

One thing that is useful to check is outstanding pull requests, @gndclouds fixed a merge conflict that allowed @martinalong to merge after checking the work.